include/envoy/event/file_event.h

```diff
@@ -18,11 +18,12 @@ struct FileReadyType {
   static const uint32_t Closed = 0x4;
 };
 
-#define FORCE_LEVEL_EVENTS
 enum class FileTriggerType { Level, Edge, EmulatedEdge };
 
 static constexpr bool optimizeLevelEvents = true;
```

Comment:
I suggest making optimizeLevelEvents an #ifdef so it's possible to get the desired behavior by passing in extra flags to the bazel build/test invocation.

Reply:
fixed in 342fe7383e53537121d9003b2731bda5640f2da8